graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -1,8 +1,9 @@
 # Copyright 2019-present Kensho Technologies, LLC.
 from collections import namedtuple
-from copy import deepcopy
+from copy import copy
 
 from graphql import build_ast_schema
+from graphql.language import ast as ast_types
```

Comment:
Good point, I imported this way out of habit because I did the same thing for files where I use many different AST components. I'll import just `Name` here.